src/common/strvararg.cpp

```diff
@@ -703,52 +703,55 @@ wxFormatString::ArgumentType DoGetArgumentType(const CharType *format,
         return wxFormatString::Arg_Unused;
     }
 
-    wxCHECK_MSG( parser.pspec[n-1] != NULL, wxFormatString::Arg_Unknown,
-                 "requested argument not found - invalid format string?" );
-
-    switch ( parser.pspec[n-1]->m_type )
+    if (0<n)
```

Comment:
Wouldn't this be better as a `wxCHECK_MSG` at the beginning of this function? It would avoid all the blank changes.
Apart from that:
- `n` is always positive, so comparing it with `0` in the condition is not needed;
- `if` spacing is improper, see the previous condition at line 694.